src/components/ExportDataModal.js

```diff
@@ -0,0 +1,228 @@
+import _ from 'lodash/fp'
+import PropTypes from 'prop-types'
+import { b, div, h } from 'react-hyperscript-helpers'
+import { buttonPrimary, spinnerOverlay } from 'src/components/common'
+import ErrorView from 'src/components/ErrorView'
+import { icon } from 'src/components/icons'
+import Modal from 'src/components/Modal'
+import { withWorkspaces, WorkspaceSelector } from 'src/components/workspace-utils'
+import { ajaxCaller } from 'src/libs/ajax'
+import colors from 'src/libs/colors'
+import { reportError } from 'src/libs/error'
+import { requiredFormLabel, formLabel } from 'src/libs/forms'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+import validate from 'validate.js'
+
+
+class infoTile extends Component {
```

Comment:
This is a good abstraction. A few nits:
Since it's stateless, you could use the 'functional' component notation which is slightly more compact.
PropTypes are probably overkill for something simple and file-local like this.
Component names should generally be title-cased, e.g. `InfoTile`